frontend/packages/dev-console/src/components/formik-fields/RadioButtonField.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { useField } from 'formik';
+import { FormGroup, Radio } from '@patternfly/react-core';
+import { RadioButtonProps } from './field-types';
+import { getFieldId } from './field-utils';
+
+const RadioButtonField: React.FC<RadioButtonProps> = ({
+  label,
+  value,
+  checked,
+  helpText,
+  required,
+  ...props
+}) => {
+  const [field, { touched, error }] = useField(props.name);
+  const fieldId = getFieldId(value, 'radiobutton');
```

Comment:
@nemesis09 You would probably want to still add the name part in the id because you might have same values in a form with two different sets of radio buttons, in that case the name part would be unique.

```suggestion
  const fieldId = getFieldId(`${name}-${value}`, 'radio-button');
```

Reply:
Yes. I've updated the component to support that.